src/components/DuetRoom.tsx

```diff
@@ -31,55 +33,36 @@ const useStyles = makeStyles(theme => ({
   },
   box: {
     flexGrow: 100,
+    position: 'relative',
   },
   header: {
     flexGrow: 0,
   },
   piano: {
     flexGrow: 0,
   },
+  readyButton: {
```

Comment:
Should we remove the absolute spacing in the future? If so can we put a TODO here